matrix/instrumentations.yaml

```diff
@@ -0,0 +1,303 @@
+- keys:
+    - ASPNET
+  library: ASP.NET Framework (.NET Framework)
+  version: See :ref:`dotnet-otel-versions`
+  comment: MVC / WebApi (Only integrated pipeline mode supported). Metrics requires trace instrumentation.
+  stability: beta
+  supportabilityLevel: community
```

Comment:
```suggestion
  supportability: community
```

Reply:
Fabrizio? what is your preference to the naming?

Reply:
`support`

Reply:
2fa2ba84f283eca1f137d26fb657cf321bbad4f1